src/app/MessageDef/WriteRequest.h

```diff
@@ -148,6 +148,14 @@ class Builder : public chip::app::Builder
      *  @return A reference to *this
      */
     WriteRequest::Builder & MoreChunkedMessages(const bool aMoreChunkedMessages);
+
+    /**
+     *  @brief Get reference to AttributeDataList::Builder
```

Comment:
Please remove redundant comments or rewrite to complement the code rather than re-stating it in comments. This seems to occur more times below. Please clean all instances up.

Reply:
removed, thanks